packages/ens-utils/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@namehash/ens-utils",
   "description": "",
-  "version": "0.0.0-development",
+  "version": "1.0.0",
```

Comment:
Hmm. I was hoping to make the v1.0 release more symbolic once we reach a special milestone of maturity. What do you think? Appreciate your advice. If there's a practical reason to update this to v1.0 now then happy to proceed.

Reply:
@lightwalker-eth I'm not sure how or why but it looks like someone published the package manually and it's created the `0.0.0-development` release on NPM. https://www.npmjs.com/package/@namehash/ens-utils?activeTab=versions

Because of this, I will now start the count from `1.12.0` and update this PR.